pipeline-runner/R/qc-7-embed_and_cluster.R

```diff
@@ -14,3 +24,37 @@ embed_and_cluster <- function(scdata, config, sample_id, task_name = 'configureE
 
   return(result)
 }
+
+format_cell_sets_object <- function (cell_sets,type,color_pool) {
+  key <- type
```

Comment:
what is the point of creating a variable `key` here? You can just use `type` directly, or at least this is how it look like